extra/docs_landing.html

```diff
@@ -48,335 +48,195 @@
 </style>
 
 <div class="legalnotice"></div>
-
-<div class="row my-4">
-  <div class="col-md-6 col-12">
-    <p></p>
-    <h2>Learn about Elastic</h2>
-    <p>
-      Elastic provides flexible search, monitoring, and security solutions based on Elasticsearch.
-    </p>
-    <p>
-      <a class="inline-block mr-3" href="https://www.elastic.co/guide/index.html#viewall">All Elastic docs</a>
-      <a class="inline-block mr-3" href="https://www.elastic.co/guide/en/starting-with-the-elasticsearch-platform-and-its-solutions/current/new.html">Release docs</a>
-      <a class="inline-block mr-3" href="https://www.elastic.co/videos">Videos & Webinars</a>
-   </p>
-   </div>
-   <div class="col-md-6 col-12">
-     <div class="embed-responsive embed-responsive-16by9">
-       <iframe class="embed-responsive-item vidyard_iframe" src="https://play.vidyard.com/qAS1W24928bBrc2sv5f5Ti?v=4.3" scrolling="no" frameborder="0" allowfullscreen=""></iframe>
-     </div>
-   </div>
+<div>
+  <p>
+    <a class="inline-block mr-3" href="https://www.elastic.co/guide/index.html#viewall">Browse all docs</a>
+    <a class="inline-block mr-3" href="en/starting-with-the-elasticsearch-platform-and-its-solutions/current/new.html">Release docs</a>
+  </p>
+</div>
+<div id="capability-tabs-container" class="tabs" data-tab-group="doc-types">
+  <div id="button-container" role="tablist" aria-label="Documentation groups" class="clearfix">
+    <button id="solution-tab" role="tab" aria-selected="true" aria-controls="solution-tab-contents" class="float-lg-left d-block d-lg-inline tab-button active">Solutions</button>
+    <button id="product-tab" role="tab" aria-selected="false" aria-controls="product-tab-contents" class="float-lg-left d-block d-lg-inline tab-button">Products</button>
+    <button id="deployment-tab" role="tab" aria-selected="false" aria-controls="deployment-tab-contents" class="float-lg-left d-block d-lg-inline tab-button">Deploy anywhere</button>
```

Comment:
IMO "Deploy anywhere" over promises and might be misleading. It's also not parallel with the other tab names. 

What about something like Deployment options? Deployment choices?

Reply:
Incidentally as a user I think I would expect to see the serverless link on this tab.

Reply:
Change to `Deploy`

Reply:
Relates to https://github.com/elastic/docs/pull/2805#discussion_r1389209903

Reply:
Renamed to "Deploy" in dcdc4c2e861b968b1068f71f2674217dda23cfc8